### DISCUSSION TITLE
Does USER.conf already implemented yet or not?

### DISCUSSION THREAD

**Discussion by benyaminl** [rpm-software-management/dnf5#546] [Q&A]
Hello, I already seen https://github.com/rpm-software-management/dnf5/issues/131, but does that mean /etc/dnf/dnf5-aliases.d/USER.conf hasn't been implemented yet? I want to copy paste my /etc/dnf/aliases.d/USER.conf, but got error

USER.conf
```
[main]
enabled = True

[aliases]
provides = "repoquery --provides"
```

the output 
```
sudo dnf5 provides /etc/dnf/dnf.conf
Syntax error in file "/etc/dnf/dnf5-aliases.d/USER.conf" on line 2
Unknown argument "provides" for command "dnf5". Add "--help" for more information about the arguments.
```

**Comment by j-mracek** [comment]
it looks like that you use aliases syntax from DNF5. I was unable to find a documentation in out upstream but look et https://github.com/rpm-software-management/dnf5/blob/main/dnf5/config/usr/share/dnf5/aliases.d/compatibility.conf.

You can try:
```
['provides']
type = 'command'
attached_command = 'repoquery'
descr = "Alias for 'repoquery --provides'"
group_id = 'user-aliases'
complete = true
attached_named_args = [
 { id_path = 'repoquery.provides' }
]
```
